packages/plugins/graphql/server/services/content-api/wrap-resolvers.js

```diff
@@ -31,87 +31,94 @@ const wrapResolvers = ({ schema, strapi, extension = {} }) => {
   // Fields filters
   const isValidFieldName = ([field]) => !field.startsWith('__');
 
-  const typeMap = schema.getTypeMap();
-
-  // Iterate over every field from every type within the
-  // schema's type map and wrap its resolve attribute if needed
-  Object.entries(typeMap).forEach(([type, definition]) => {
-    const isGraphQLObjectType = definition instanceof GraphQLObjectType;
-    const isIgnoredType = introspectionQueries.includes(type);
-
-    if (!isGraphQLObjectType || isIgnoredType) {
-      return;
-    }
-
-    const fields = definition.getFields();
-    const fieldsToProcess = Object.entries(fields).filter(isValidFieldName);
-
-    for (const [fieldName, fieldDefinition] of fieldsToProcess) {
-      const defaultResolver = get(fieldName);
-
-      const path = `${type}.${fieldName}`;
-      const resolverConfig = getOr({}, path, resolversConfig);
-
-      const { resolve: baseResolver = defaultResolver } = fieldDefinition;
-
-      // Parse & initialize the middlewares
-      const middlewares = parseMiddlewares(resolverConfig, strapi);
-
-      // Generate the policy middleware
-      const policyMiddleware = createPoliciesMiddleware(resolverConfig, { strapi });
-
-      // Add the policyMiddleware at the end of the middlewares collection
-      middlewares.push(policyMiddleware);
-
-      // Bind every middleware to the next one
-      const boundMiddlewares = middlewares.map((middleware, index, collection) => {
-        return (...args) =>
-          middleware(
-            // Make sure the last middleware in the list calls the baseResolver
-            index >= collection.length - 1 ? baseResolver : boundMiddlewares[index + 1],
-            ...args
-          );
-      });
-
-      /**
-       * GraphQL authorization flow
-       * @param {object} context
-       * @return {Promise<void>}
-       */
-      const authorize = async ({ context }) => {
-        const authConfig = get('auth', resolverConfig);
-        const authContext = get('state.auth', context);
-
-        const isMutationOrQuery = ['Mutation', 'Query'].includes(type);
-        const hasConfig = !isNil(authConfig);
-
-        const isAuthDisabled = authConfig === false;
-
-        if ((isMutationOrQuery || hasConfig) && !isAuthDisabled) {
-          try {
-            await strapi.auth.verify(authContext, authConfig);
-          } catch (error) {
-            throw new ForbiddenError();
+  const typesMaps = [schema.getTypeMap()];
```

Comment:
as the subscription is a type are you sure it's not in this already ?

Reply:
Good catch